### ISSUE TITLE
Rendering Problems

### ISSUE THREAD

**Issue by sky-gh** [htdvisser/hugo-base16-theme#14]
Trying to set this theme up but it's behaviour is erratic with regards to proper rendering. For example: Theme demo on its [gohugo page](https://themes.gohugo.io/theme/base16/) vs [github page](https://htdvisser.github.io/hugo-base16-theme/). Problem described in [hugo-discourse](https://discourse.gohugo.io/t/theme-rendered-broken/7434/4).

**Comment by paskal** [comment]
Another example of improper render:

    ``` json
    {
        "shell":true,
        "cmd": ["rsync -az '$file' server.example.org:~ && ssh server.example.org 'chmod +x ./$file_name; ./$file_name'"],
    }
    ```

Converts to:
![image](https://user-images.githubusercontent.com/712534/37653503-37d93238-2c50-11e8-84ca-eae09d75a215.png)
♬ The Doors - Break On Through (To The Other Side)

**Comment by htdvisser** [comment]
[themes.gohugo.io/theme/base16](https://themes.gohugo.io/theme/base16/) looks fine now:

![screen shot 2018-04-01 at 19 03 46](https://user-images.githubusercontent.com/181308/38175861-f8ebec62-35e3-11e8-93fe-7df087c06526.png)

Overflow of code block reported by @paskal should be fixed by 04efce37f3dc1f59ec7f2d1ad9ea9027f36c0793.
